.github/scripts/in_develop_labeler.py

```diff
@@ -4,8 +4,12 @@
 def main():
 	args = arg_getters.get_in_develop_labeler_args()
 	try:
+		#get pr body
+		pr_body = api_handler.get_pr_by_number(args.token, args.pr_num)["body"]
```

Comment:
Shouldn't this be `pr_num`?

Reply:
hmm, no I don't think so. 
It gets the `pr_body` using the `get_pr_by_number()` function from the `pr_num` argument (`args.pr_num`), and then choosing the body (`["body"]`)

the pr_body is later used to check for checking if any issues are linked to the PR